src/v1/destinations/iterable/networkHandler.ts

```diff
@@ -0,0 +1,38 @@
+import { prepareProxyRequest, proxyRequest } from '../../../adapters/network';
+import { processAxiosResponse } from '../../../adapters/utils/networkUtils';
+import { BULK_ENDPOINTS } from '../../../v0/destinations/iterable/config';
+import { GenericStrategy } from './strategies/generic';
+import { TrackIdentifyStrategy } from './strategies/track-identify';
+
+type ResponseParams = {
+  destinationRequest: {
+    endpoint: string;
+  };
+};
+
```

Comment:
If needed we can have local types like this

Reply:
done